core/drivers/fabric-driver/server/listener.ts

```diff
@@ -291,9 +281,26 @@ const unregisterListenerForEventSubscription = async (
  * Load event subscriptions from the driver database.
  * Start a listener for each channel from which one of the subscribed events originate.
  **/
-const loadEventSubscriptionsFromStorage = (
-): void => {
-    // TODO
+const loadEventSubscriptionsFromStorage = async (networkName: string): Promise<boolean> => {
+    console.log('Starting event listeners for subscribed events in database...')
+    try {
+        const eventMatchers = await readAllEventMatchers();
+        for (const eventMatcher of eventMatchers) {
+            try {
+                const listenerHandle = await registerListenerForEventSubscription(
+                    eventMatcher,
+                    networkName
```

Comment:
Referring to my earlier comment: I suggest you pass the database type and name as parameter here.